pkg/volume/azuredd/attacher.go

```diff
@@ -72,23 +72,23 @@ func (a *azureDiskAttacher) Attach(spec *volume.Spec, nodeName types.NodeName) (
 	lun, err := diskController.GetDiskLun(volumeSource.DiskName, volumeSource.DataDiskURI, nodeName)
 	if err == cloudprovider.InstanceNotFound {
 		// Log error and continue with attach
-		klog.Warningf(
-			"Error checking if volume is already attached to current node (%q). Will continue and try attach anyway. err=%v",
-			nodeName, err)
+		klog.InfoS(
+			"Error checking if volume is already attached to current node. Will continue and try attach anyway",
+			"nodeName", nodeName, "err", err)
 	}
 
 	if err == nil {
 		// Volume is already attached to node.
-		klog.V(2).Infof("Attach operation is successful. volume %q is already attached to node %q at lun %d.", volumeSource.DiskName, nodeName, lun)
+		klog.V(2).InfoS("Attach operation is successful. Volume is already attached to node at lun.", "volumeDiskName", volumeSource.DiskName, "nodeName", nodeName, "lunNumber", lun)
 	} else {
-		klog.V(2).Infof("GetDiskLun returned: %v. Initiating attaching volume %q to node %q.", err, volumeSource.DataDiskURI, nodeName)
+		klog.V(2).InfoS("Initiating attaching volume to node. GetDiskLun returned error.", "volumeDiskURI", volumeSource.DataDiskURI, "nodeName", nodeName, "err", err)
```

Comment:
```suggestion
		klog.V(2).InfoS("Initiating attaching volume to node. GetDiskLun returned error.", "URI", volumeSource.DataDiskURI, "node", klog.KRef("", nodeName), "err", err)
```

Reply:
The above problems have been modified, waiting for your review with pleasure. @serathius